src/pages/access_review/VoteAsChair.js

```diff
@@ -62,8 +62,9 @@ export const VoteAsChair = hh(class VoteAsChair extends React.PureComponent {
   };
 
   render() {
-    const { vote, rpVote, onUpdate, matchData, accessElectionOpen, rpElectionOpen, hasLibraryCard } = this.props;
-    const errorMessage = "The Researcher must have a Library Card before this DAR can be approved.";
+    const { vote, rpVote, onUpdate, matchData, accessElectionOpen, rpElectionOpen, libraryCards } = this.props;
+    const hasLibraryCard = !isNil(head(libraryCards));
+    const errorMessage = "The Researcher must have a Library Card before this DAR can be approved. You can still deny the request or vote on question 2.";
```

Comment:
Minor suggestion, and then the code is good to go.

With your follow up sentence I realized we can make that old first sentence a bit more specific. Something like ```The Researcher must have a Library Card before data access can be granted. You can still deny this request and/or vote on question 2```